client/cli/src/lib.rs

```diff
@@ -253,7 +253,7 @@ pub fn init_logger(pattern: &str) {
 					format!("{}", Colour::Blue.bold().paint(x))
 				});
 			let millis = (now.tm_nsec as f32 / 1000000.0).floor() as usize;
-			let timestamp = format!("{}.{}", timestamp, millis);
+			let timestamp = format!("{}.{:03}", timestamp, millis);
```

Comment:
In recent versions of chrono it seems there is something to get the milliseconds already. https://docs.rs/chrono/0.4.11/chrono/format/strftime/index.html `%.3f`

Reply:
Ah yes it's time and not chrono. But I did introduced chrono so it's in the dependencies already. I think we can remove time since it doesn't provide the milliseconds.

Reply:
I'd just fix this simple bug